src/custom/theme/baseTheme.tsx

```diff
@@ -109,16 +113,15 @@ export function colors(darkMode: boolean): Colors {
   }
 }
 
-export function themeVariables(darkMode: boolean, shouldBlurBackground: boolean, colorsTheme: Colors) {
-  const background = cowSwapBackground(darkMode, shouldBlurBackground)
-
+export function themeVariables(darkMode: boolean, colorsTheme: Colors) {
   return {
     body: {
       background: css`
-        background: url(data:image/svg+xml;base64,${background}) no-repeat 100% / cover fixed,
-          ${darkMode
-            ? 'linear-gradient(180deg,rgba(20, 45, 78, 1) 10%, rgba(22, 58, 100, 1) 30%)'
-            : 'linear-gradient(180deg,rgba(164, 211, 227, 1) 5%, rgba(255, 255, 255, 1) 40%)'};
+        ${darkMode
+          ? `
+          background-color: ${colorsTheme.blueDark1};
+          background-image: radial-gradient(50% 500px at 50% -13%,hsl(216deg 100% 20% / 70%) 0%,#071832 50%, #06162d 100%),radial-gradient(circle at -70% 50%,hsla(215,100%,20%,0.7) 0,transparent 50%);`
```

Comment:
Nitpick: add space after commas to improve readability and reduce chance of error

Reply:
Updated 👍🏼